internal/services/authorization/client_config_data_source.go

```diff
@@ -45,7 +45,7 @@ func dataSourceArmClientConfigRead(d *pluginsdk.ResourceData, meta interface{})
 	_, cancel := timeouts.ForRead(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	d.SetId(time.Now().UTC().String())
+	d.SetId("armClientConfig")
```

Comment:
can we make this a meaningful value? like
```suggestion
	d.SetId("clientConfig/$subscriptionid/$clientid/$objectid")
```